### PR TITLE
Implement spinner for search and spinner for login

### PR DESCRIPTION
**Pull request by aramis-eb**
implements two spinners: one to login and the other for a search

### CODE REVIEW

File: static/js/loginManager.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const LoginManager = (function() {
```
Comment (julianrofrano-eb) [inline static/js/loginManager.js:1]: this should be in camelCase starting with lower case

File: static/js/formManager.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const FormManager = (function() {
```
Comment (julianrofrano-eb) [inline static/js/formManager.js:1]: this should be in camelCase starting with lower case

File: static/js/loginManager.js (hunk 1 above)
Comment (aramis-eb) [reply static/js/loginManager.js:1]: Resolved!

File: static/js/formManager.js (hunk 2 above)
Comment (aramis-eb) [reply static/js/formManager.js:1]: Resolved!

**Pull request merged**